generators/app/about-us/about-us-ng-template-writer.js

```diff
@@ -16,22 +16,30 @@ function write(generator) {
     generator.nativeLanguage = jhipsterAppConfig.nativeLanguage;
     generator.useSass = jhipsterAppConfig.useSass;
 
-    const s = generator.navElementKey.trim().replace(' ', '-').replace('_', '-');
+    const s = generator.navElementKey
+        .trim()
+        .replace(' ', '-')
+        .replace('_', '-');
     generator.translationKeyMenu = _.kebabCase(s).toLowerCase();
     generator.componentStartCase = _.startCase(s);
 
-    const prefix = jhipsterAppConfig.jhiPrefix ? `${_.kebabCase(jhipsterAppConfig.jhiPrefix)}-` : '';
+    const prefix = jhipsterAppConfig.jhiPrefix
+        ? `${_.kebabCase(jhipsterAppConfig.jhiPrefix)}-`
```

Comment:
please revert any formatting changes (in all files). I don't think I like this format.

Reply:
I have understood that you don't like a formatter.. It was useful to visualize better files and code (and it is also used bu the generator-jhipster). But if you don't want, no problem :)

Reply:
Like I said this project supports projects built with JHipster.
However, the coding guidelines and other aspects of this project are independent of JHipster project.

Reply:
> 
> 
> > I have split in PR #37 and #38
> 
> Kindly give descriptions to your PRs - it helps the reviewer (me) to identify what changes are made for.

The commit message refers to the issue. It is the solution, so it is already an explanation.